composer.json

```diff
@@ -17,7 +17,7 @@
     },
     "require": {
         "neos/neos-development-collection": "2.0.x-dev",
-        "neos/flow-development-collection": "@dev",
+        "neos/flow-development-collection": "3.0.x-dev",
```

Comment:
Did you test that? Because i freshly created/cloned a lts and got dev-master for 2.3?! But gonna try this again. Maybe the dep Inside the dev-collection is/was also broken